cdktf.json

```diff
@@ -1,12 +1,11 @@
 {
   "language": "typescript",
-  "app": "npm run --silent compile && node main.js",
-  "terraformProviders": [
-    "hashicorp/local"
-  ],
+  "app": "npx ts-node main.ts",
+  "terraformProviders": [],
   "terraformModules": [],
   "context": {
     "excludeStackIdFromLogicalIds": "true",
-"allowSepCharsInLogicalIds": "true"
-  }
+    "allowSepCharsInLogicalIds": "true"
+  },
+  "projectId": "62bfd60a-3f06-4709-b2ba-6c4ece844fbc"
```

Comment:
I'm wondering whether we should keep this one out of this project so that a new one is generated if users use this project as a starting point?

Reply:
I don't think folks will copy this 1:1 and use it as their own, it's too specific for that. I'd rather imagine a `cdktf init` and looking at this as an example, copying code parts

Reply:
Your call 😄 I don't have a strong opinion here.